src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,10 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.map(comment => (
+      <CommentInfo comment={comment} key={comment.id} />
+    ))}
+  </div>
```

Comment:
It will be a good practice to use <ul> and <li> in the mapping process. The name of the class tells us)

A standard example using map in React 

```
<ul>
    {userNames.map(name => <li>{name}</li>)}
 </ul>
```

See more [here](https://www.shecodes.io/athena/7184-how-to-use-the-map-method-in-react)